packages/riipen-ui/package.json

```diff
@@ -46,6 +46,8 @@
     ]
   },
   "peerDependencies": {
+    "draft-convert": "^2.1.10",
+    "draft-js": "^0.11.6",
```

Comment:
This was the real fix I was going for. Basically we ended up having 2 version of `draft-js` and `draft-convert` in `admin-web` and `platform-web` which was causing a ton of issues.